src/main/java/org/jboss/windup/operator/controller/WindupController.java

```diff
@@ -0,0 +1,79 @@
+package org.jboss.windup.operator.controller;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import io.fabric8.kubernetes.client.Watcher;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import lombok.extern.java.Log;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jboss.windup.operator.model.WindupResource;
+import org.jboss.windup.operator.model.WindupResourceDoneable;
+import org.jboss.windup.operator.model.WindupResourceList;
+import org.jboss.windup.operator.util.WindupDeployment;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@Log
+@ApplicationScoped
+public class WindupController implements Watcher<WindupResource> {
+	@Inject
+	MixedOperation<WindupResource, WindupResourceList, WindupResourceDoneable, Resource<WindupResource, WindupResourceDoneable>> crClient;
+
+	@Named("namespace")
+	String namespace;
+
+	@ConfigProperty(name = "operator.serviceaccount", defaultValue = "windup-operator")
+	String serviceAccount;
+
+	@ConfigProperty(name = "operator.sso_public_key")
+	String ssoPublicKey;
+
+	@Inject
+	KubernetesClient k8sClient;
+
+	private void onAdd(WindupResource resource) {
+		log.info("Event ADD " + resource.getMetadata().getName());
+		if (!resource.isDeploying() && !resource.isReady()) {
+			new WindupDeployment(resource, crClient, k8sClient, namespace, serviceAccount, ssoPublicKey).deploy();
+		}
+	}
+
+	private void onUpdate(WindupResource newResource) {
+		log.info("Event UPDATE " + newResource.getMetadata().getName() + " - DR "
```

Comment:
We've been looking at this log during the tests and it was not immediately clear which were the meanings for `DR` and `RD`.
Could you please make them more verbose? removing the acronyms would be help for sure :+1: